lib/0001_retrieve-secrets.sh

```diff
@@ -77,7 +77,7 @@ def generate_vcap_creds_hash
     exit(1)
   end
 
-  @creds_map.reject! { |key, _| creds[key.to_s].empty? }
+  @creds_map.reject! { |key, _| creds[key.to_s].to_s.empty? }
```

Comment:
I know it's a bit more verbose but this would be a bit more explicit:

`creds[key.to_s].nil? || creds[key.to_s].empty?`

It's a little roundabout to use a `to_s` cast to make sure you're checking `nil`.

Reply:
I'm not actually checking nil, the issue was that the `creds[key.to_s]` value was Fixnum, which didn't have an `empty?` method